GameMod/GameMod.cs

```diff
@@ -101,45 +102,6 @@ public static bool HasInternetMatch()
             return GameVersion.CompareTo(new Version(1, 0, 1885)) >= 0;
         }
 
-        // add modified indicator to main menu
```

Comment:
Moved to PatchVersionInfo class which centralizes all changes relating to the olmod version display